semanticdb/metap/src/main/scala/scala/meta/cli/Metap.scala

```diff
@@ -33,4 +34,12 @@ object Metap {
     val main = new Main(settings, reporter)
     main.process()
   }
+
+  def print(doc: TextDocument): String = {
```

Comment:
This method does not belong to the public API when I think of it, `TextDocument` is internal so it cannot be part of a public signature. That is why I added this method to internal API. wdyt @xeno-by ?

Reply:
If you need this method only for tests, I'd just move it to tests. If you plan to use it right away somewhere else, I'd put it in `scala.meta.internal.semanticdb3.Print`.